Header                          header

geometry_msgs/PoseStamped[]     base_footprint_plan
geometry_msgs/PoseStamped[]     base_footprint_alternative_plan
geometry_msgs/PoseStamped       goal_base_footprint
int32                           path_choice
masterarbeit/Intersect          intersect
geometry_msgs/Point[]           transformed_circles_centers
geometry_msgs/Vector3[]         transformed_circles_velocities
float32                         odom_vel
float32                         main_path_length
float32                         alternative_path_length
bool                            direction
float64                         realtime_duration